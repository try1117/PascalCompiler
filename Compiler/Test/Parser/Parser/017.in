program test;
var
  a : integer = not 0;
const
  b : double = not(a - 1);
  c : integer = not a - 5;
  d : char = a + -1 + 5 + '0' + 2;

begin
end.
