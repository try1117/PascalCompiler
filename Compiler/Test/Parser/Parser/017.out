test : function()
   resultType : Nil

test declarations:
   a : Integer
   |-- 1

   b : Const Double
   |-- 1.000000

   c : Const Integer
   |-- -5

   d : Const Char
   |-- 7

